packages/regenesis-surgery/test/setup.ts

```diff
@@ -76,6 +96,35 @@ class TestEnv {
     if (this.surgeryDataSources === undefined) {
       this.surgeryDataSources = await loadSurgeryData()
 
+      if (!this.preL2Provider) {
+        console.log('Initializing pre GenesisJsonProvider...')
+        // Convert the genesis dump into a genesis file format
+        const genesis = { ...this.surgeryDataSources.genesis }
+        for (const account of this.surgeryDataSources.genesisDump) {
```

Comment:
should this be iterating over `this.surgeryDataSources.dump` instead, since the state dump is the one with the accounts?